crates/sui-bridge/src/server/handler.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::BridgeError;
+use crate::eth_client::EthClient;
+use crate::sui_client::SuiClient;
+use crate::types::SignedBridgeAction;
+use async_trait::async_trait;
+use axum::Json;
+use sui_sdk::SuiClient as SuiSdkClient;
+
+#[async_trait]
+pub trait BridgeRequestHandlerTrait {
+    async fn handle_eth_tx_hash(
```

Comment:
Comment on what each handler does?

Reply:
will do in a later PR

Reply:
https://github.com/MystenLabs/sui/pull/15290/files#diff-1489047ec8e9221e8c8c0990e62aa663f06fa60dbbece5df6d1fd7e4a4ebac99R14-R24